addons/mail/static/src/components/chatter/chatter_tests.js

```diff
@@ -4,6 +4,7 @@ odoo.define('mail/static/src/components/chatter/chatter_tests', function (requir
 const components = {
     Chatter: require('mail/static/src/components/chatter/chatter.js'),
 };
+const { makeDeferred } = require('mail/static/src/utils/deferred/deferred.js');
```

Comment:
diff not used, chatter container not covered by a test actually?

Reply:
Attempt making a test for chatter but it's not doable at chatter level. Added tests at form_renderer level.